src/Resources/Resources/help/Test-AzResourceGroupDeployment.md

```diff
@@ -144,6 +153,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -IgnoreDynamicParameters
+Skip resolving dynamic parameters; this should be used when running a script or running this command as a job.
```

Comment:
Not quite.  This should be used if all template parameters are satisfied using a templateparamaterobject or templateparameterfile

Reply:
Same comment -- updated parameter name and description.